### PR TITLE
NTR humeral circumflex arteries

### PR DESCRIPTION
**Pull request by aleixpuigb**
Related to #3277, adds anterior and posterior circumflex humeral arteries. These terms are part of the VCCF and have been added using a DOSDP pattern.

### CODE REVIEW

File: src/patterns/data/default/artery_and_arteriole_pattern.tsv [hunk 1]
```diff
@@ -18,4 +18,6 @@ UBERON:8600098	dorsal lingual artery	FMA:49546	UBERON:0001637	UBERON:0001456	"An
...
\ No newline at end of file
+UBERON:8600102	left anterior descending artery	FMA:3862	UBERON:0001637	UBERON:0000915	"An artery that branches from the left coronary artery and supplies the anterior wall of the left ventricle, the interventricular septum, and the apex of the heart."	wikipedia:Left_anterior_descending_artery	anterior interventricular branch of left coronary artery	FMA:3862	UBERON:0001626		https://orcid.org/0000-0001-6677-8489	2024-06-06T09:53:48Z
+UBERON:8600106	anterior humeral circumflex artery	FMA:22680	UBERON:0001637	UBERON:0001467	"An artery that branches from the axilliary artery and supplies the deltoid, biceps brachii, and coracobrachialis muscles, the glenohumeral joint, the proximal humeral head, and the skin overlying the anterior aspect of the proximal humerus. This artery courses laterally across the anterior surface of the humerus and anastomoses with the posterior humeral circumflex artery."	https://www.elsevier.com/resources/anatomy/cardiovascular-system/arteries/anterior-circumflex-humeral-artery/20091			UBERON:0001394	UBERON:0001476|UBERON:0001507|UBERON:0001505|UBERON:0001470|UBERON:0006801	https://orcid.org/0000-0001-6677-8489	2024-08-20T09:53:48Z
```
Comment (rays22) [inline src/patterns/data/default/artery_and_arteriole_pattern.tsv:22]: You say in the definition: "supplies ... the skin overlying the anterior aspect of the proximal humerus".
Should we add the [upper arm skin](http://purl.obolibrary.org/obo/UBERON_0004263) to the `vessel_supplies_blood_to` list?

You might want to add `arteria circumflexa anterior humeri` (`synonym_xrefs`: `https://ta2viewer.openanatomy.org/?id=4630`).

File: src/patterns/data/default/artery_and_arteriole_pattern.tsv [hunk 2]
```diff
@@ -18,4 +18,6 @@ UBERON:8600098	dorsal lingual artery	FMA:49546	UBERON:0001637	UBERON:0001456	"An
...
+UBERON:8600107	posterior humeral circumflex artery	FMA:22684	UBERON:0001637	UBERON:0001467	"An artery that branches from the axilliary artery and supplies the deltoid, teres major, teres minor, and long and lateral heads of the triceps brachii muscles, glenohumeral joint, and the cutaneous supply to the area posterolateral to the proximal humerus. This artery courses posteriorly with the axillary nerve, through the quadrangular space, to the posterior aspect of the humerus and anastomoses with the anterior humeral circumflex artery."	https://www.elsevier.com/resources/anatomy/cardiovascular-system/arteries/posterior-circumflex-humeral-artery/24676			UBERON:0001394	UBERON:0001476|UBERON:0001478|UBERON:0010496|UBERON:0001509|UBERON:0001470	https://orcid.org/0000-0001-6677-8489	2024-08-20T09:53:48Z
```
Comment (rays22) [inline src/patterns/data/default/artery_and_arteriole_pattern.tsv:23]: You say in the definition: "supplies ... the cutaneous supply to the area posterolateral to the proximal humerus".
Does that mean that the list `vessel_supplies_blood_to` should also include the posterolateral skin of the shoulder, or the closest term in Uberon, [upper arm skin](http://purl.obolibrary.org/obo/UBERON_0004263)?

You might want to add `arteria circumflexa posterior humeri` as a synonym (`synonym_xrefs`: `https://ta2viewer.openanatomy.org/?id=4631`).

**Review by rays22 — CHANGES_REQUESTED** [review]
The filler data for the new terms look very nice. I have just some minor suggestions for additions. Feel free to ignore them if you disagree.

File: src/patterns/data/default/artery_and_arteriole_pattern.tsv (hunk 1 above)
Comment (aleixpuigb) [reply src/patterns/data/default/artery_and_arteriole_pattern.tsv:22]: > You say in the definition: "supplies ... the skin overlying the anterior aspect of the proximal humerus".
Should we add the [upper arm skin](http://purl.obolibrary.org/obo/UBERON_0004263) to the vessel_supplies_blood_to list?

Thank you, I will add that term to be 'supplied'.

> You might want to add arteria circumflexa anterior humeri (synonym_xrefs: https://ta2viewer.openanatomy.org/?id=4630).

I was thinking about it, but the reason for not adding it was that it should have an annotation property 'has synonym type' latin, and I thought that would overcomplicate the DOSDP.

File: src/patterns/data/default/artery_and_arteriole_pattern.tsv (hunk 2 above)
Comment (aleixpuigb) [reply src/patterns/data/default/artery_and_arteriole_pattern.tsv:23]: Thank you for your comment, same answer as in the previous comment.

**Review by rays22 — APPROVED** [review]

**Pull request merged**